modules/architecture-rhcos-updating-bootloader.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture_rhcos.adoc
+
+[id="architecture-rhcos-updating-bootloader.adoc_{context}"]
```

Comment:
The architecture section isn't supposed to have concrete procedural information. All of the task-like entities in this section are meant for purely information purposes, not to actually *do* anything. I think this information is in the wrong place.

Reply:
Thanks, @kalexand-rh. I wondered about this. I initially added the procedure in the RHCOS architecture docs because it follows a couple of pseudo-procedural sections ("Viewing Ignition configuration files" and "Changing Ignition configs after installation") I was unaware that the arch section shouldn't have procedures, although I can see why that should be the case.

Based on the [note](https://github.com/openshift/openshift-docs/pull/28071/files#diff-fa7191a8c6a55b5de8a3aa4e58352ec544b4bf4ae31dbdc650234510c673d973R12-R15) elsewhere in this PR that `bootupd` should only be used for metal and hypervisor installs, I am moving its procedure module to the bare metal install docs and to the vSphere UPI docs as part of (but towards the end of) their respective RHCOS section.

I'll be curious to know what you and @codyhoag think of this approach.

Reply:
I've also removed many of the hard wraps that I encountered to bring up to current OSDOCS guidelines.

Reply:
If the boot loader is only relevant for vSphere and bare metal UPI, I don't see any issues including them in the sections you've outlined. :+1:

Reply:
> If the boot loader is only relevant for vSphere and bare metal UPI, I don't see any issues including them in the sections you've outlined. 👍

@ashcrow Does it make sense to you if I move the bootupd install procedure to the vSphere and bare metal UPI docs?

Reply:
It makes sense to me @bobfuru.